kibana-reports/server/routes/utils/visual_report/visualReportHelper.ts

```diff
@@ -33,7 +34,8 @@ export const createVisualReport = async (
   reportParams: ReportParamsSchemaType,
   queryUrl: string,
   logger: Logger,
-  cookie?: SetCookie
+  cookie?: SetCookie,
+  chromiumPath = CHROMIUM_PATH
```

Comment:
can you put the optional parameter as the last parameter? So that in the test file, I don't think you need to pass in `undefined` anymore

Reply:
I don't know where the `cookie` parameter is used, if I swap them whenever you pass in cookie you'll have to pass `undefined` or chromium path before cookie

Reply:
In your code, the chromium parameter already has an initializer which uses `CHROMIUM_PATH`, besides test, I don't think we need to pass in different value for the path

Reply:
Oh I see your point, that won't work. Never mind, ignore my comment

Reply:
BTW I tried to mock `CHROMIUM_PATH` and used `jest.requireActual` to import other constants in the test file. The constants are correct in the test file, but in `visualReportHelper.ts` only `CHROMIUM_PATH` is correct. 

That's why I used the optional parameter workaround, if you are more familiar with jest you can try to mock the constant which would be a better solution